src/realm/parser/driver.cpp

```diff
@@ -530,17 +580,12 @@ Query EqualityNode::visit(ParserDriver* drv)
             }
         }
         else if (left_type == type_Link) {
+            REALM_ASSERT(false);
```

Comment:
What's the reason for this assert? Maybe something forgotten?

Reply:
This should have been deleted. A leftover from previous version.